bimsb/packages/tainted.scm

```diff
@@ -1,6 +1,6 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2019, 2020 Ricardo Wurmus <ricardo.wurmus@mdc-berlin.de>
-;;; Copyright © 2019 Marcel Schilling <marcel.schilling@mdc-berlin.de>
+;;; Copyright © 2019, 2020, 2021 Marcel Schilling <marcel.schilling@uni-luebeck.de>
```

Comment:
Also consider adding a .mailmap file that maps the old email address used in commits to your new address.

Reply:
Makes sense. Did I understand correctly that I'd need to explicitely include every old commit? Or is there a 'global'/catchall mapping syntax as well?

Reply:
The .mailmap file would contain something like this

```
Marcel Schilling <marcel.schilling@uni-luebeck.de>
Marcel Schilling <marcel.schilling@uni-luebeck.de> <marcel.schilling@mdc-berlin.de>
```

This collates your email addresses, specifying that the uni-luebeck address is the main identity.

Reply:
I have added the `.mailcap` file as you suggested with the latest push.